From 059b0a8b1af01aa4ef417bd30affc0cdae1a185b Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Mon, 30 Sep 2013 16:28:45 +0800
Subject: [PATCH 32/70] ENGR00281822 WFD:change not all board default add this
 apk

For all our boards, if have GPU support, sink apk will default
be installed. Others not.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 WfdSink/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/WfdSink/Android.mk b/WfdSink/Android.mk
index f7aba9f..754a73f 100644
--- a/WfdSink/Android.mk
+++ b/WfdSink/Android.mk
@@ -1,3 +1,4 @@
+ifeq ($(BOARD_HAVE_VPU),true)
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -15,3 +16,4 @@ include $(BUILD_PACKAGE)
 
 # Use the folloing include to make our test apk.
 include $(call all-makefiles-under,$(LOCAL_PATH))
+endif
-- 
1.8.0

